library/proc_macro/src/bridge/server.rs

```diff
@@ -57,6 +68,7 @@ macro_rules! define_mark_types_impls {
     }),* $(,)?) => {
         impl<S: Types> Types for MarkedTypes<S> {
             $(type $name = Marked<S::$name, client::$name>;)*
+            type Symbol = Marked<S::Symbol, client::Symbol>;
```

Comment:
Could it be added to the main RPC API macro, but with no methods? (I forget what else that might generate)

Reply:
The codegen in the `api_tags` module's `rpc_encode_decode!` call will try to define a zero-variant `#[repr(u8)] enum {}` to hold enum variant tags, and adding a `#[repr]` to an enum with no variants is a build error.

https://github.com/rust-lang/rust/blob/73443a05908b0f8043659295a854295013987876/library/proc_macro/src/bridge/mod.rs#L230
https://github.com/rust-lang/rust/blob/73443a05908b0f8043659295a854295013987876/library/proc_macro/src/bridge/rpc.rs#L69

It'd not be too difficult to modify the `rpc_encode_decode!` macro to add a special case for empty enums, though it's not necessary if I end up moving the `normalize_and_validate_ident` method to `Symbol`. I'll go ahead and do that for now.